test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -285,6 +322,20 @@ public void wipeCluster() throws Exception {
         adminExecutionContext.callApi("snapshot.delete_repository", deleteSnapshotsArgs, Collections.emptyList(), Collections.emptyMap());
     }
 
+    /**
+     * @return An exclude list of indices that will not be removed in between tests.
+     */
+    protected List<String> excludeIndices() {
```

Comment:
What about `excludeIndicesFromDeletion / excludeTemplatesFromDeletion`?

If you prefer `persistentIndices / persistentTemplates` I'm fine with it too.